### DISCUSSION TITLE
Webhook

### DISCUSSION THREAD

**Discussion by utki007** [Rapptz/discord.py#7022] [Q&A] [answered]
Unable to set avatar as bot's pfp in create_webhook
It gives an error. I have tried to convert it to bytes as well.

```self.client.avatar and self.client.avatar_url doesnt work```

**Comment by lgaan** [comment, marked as answer]
`Bot.avatar` not `Bot.avatar_url` exist. Instead you'll need to supply `str(Bot.user.avatar_url)` to `avatar_url` within `Webhook.send`.

If you wish to set the webhook's image globally then you'll need to give `create_webhook` bytes. This can be `await Bot.user.avatar_url.read()`

**Reply by utki007** [reply]
![image](https://user-images.githubusercontent.com/23389768/121245056-36481580-c8bd-11eb-9299-18e23b1f3691.png)

Hi this is the error which I am getting when I try to set webhook's image globally

**Reply by lgaan** [reply]
You need to `await` `bot.user.avatar_url.read()`.

**Reply by utki007** [reply]
Thanks a lot, it works
